problem_builder/public/js/questionnaire.js

```diff
@@ -136,10 +136,12 @@ function MCQBlock(runtime, element) {
                     choiceDOM.addClass('correct');
                     choiceResultDOM.addClass('checkmark-correct icon-ok fa-check');
                     choiceResultDOM.attr('aria-label', choiceResultDOM.data('label_correct'));
+                    choiceResultDOM.append('<span class="sr-only">'+gettext("Correct")+'</span>');
                 } else {
                     choiceDOM.addClass('incorrect');
                     choiceResultDOM.addClass('checkmark-incorrect icon-exclamation fa-exclamation');
                     choiceResultDOM.attr('aria-label', choiceResultDOM.data('label_incorrect'));
+                    choiceResultDOM.append('<span class="sr-only">'+gettext("Incorrect")+'</span>');
```

Comment:
Don't we want to remove these spans for MCQ, as we are doing in `problem_builder/public/js/mentoring_with_steps.js`?

Reply:
Sure, thanks for explaining.